setup.cfg

```diff
@@ -29,8 +29,8 @@ install_requires =
     einops>=0.3,<0.7
     # fiona 1.8+ required for reading empty files
     fiona>=1.8,<2
-    # kornia 0.6.4+ required for kornia.contrib.compute_padding
-    kornia>=0.6.4,<0.7
+    # kornia 0.6.5+ required due to change in kornia.augmentation API
+    kornia>=0.6.5,<0.7
```

Comment:
In Kornia 0.6.5+, all augmentation instance methods have a new `flags` parameter. So the transforms I added won't work with Kornia 0.6.4 and older. Once we upstream these transforms to Kornia, we'll need to depend on an even newer version anyway.